src/webgpu/api/validation/createSampler.spec.ts

```diff
@@ -0,0 +1,19 @@
+export const description = `
+createSampler validation tests.
+
+Test Plan: 
+
+* lodMinClamp:
```

Comment:
I just added a new `.unimplemented()` so you can do e.g. the following:

```ts
g.test('lodMinClamp')
.desc('Test various values for lodMinClamp')
.unimplemented();
```

Could you update this to that format?

You can even pre-write the `.params()` if you want.